pkg/ottl/contexts/internal/span.go

```diff
@@ -127,6 +127,8 @@ func SpanPathGetSetter[K SpanContext](path ottl.Path[K]) (ottl.GetSetter[K], err
 			}
 		}
 		return accessStatus[K](), nil
+	case "flags":
```

Comment:
as this is a new accessible property added to the span `context`, can you please also add an entry in the [readme](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl/contexts/ottlspan)?